Makefile

```diff
@@ -1,9 +1,8 @@
-ifeq ($(SHELL), cmd)
-	VERSION := $(shell git describe --exact-match --tags 2>nil)
-	HOME := $(HOMEPATH)
-else ifeq ($(SHELL), sh.exe)
+ifeq ($(OS), Windows_NT)
 	VERSION := $(shell git describe --exact-match --tags 2>nil)
 	HOME := $(HOMEPATH)
+	CGO_ENABLED ?= 0
+	export CGO_ENABLED
```

Comment:
Switched over to disabling CGO in Windows unless explicitly enabled.  Not entirely sure this is a good idea but I'm trying to avoid the confusion of suddenly needing gcc to build.